config.ru

```diff
@@ -2,4 +2,6 @@
 
 require_relative 'config/environment'
 
+require "#{RAILS.root}/app/classes/generate_new_week_assignments.rb"
```

Comment:
Rails should `autoload` (a Rails' magical method) everything in the `/app` directory so you don't need to require it. If you did need to require it then you'd want to put this kind of line in `config/application.rb` at some point after the `Bundler.require` (It's a fun exercise to follow the boot order in that file).